src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Text.cs

```diff
@@ -165,7 +165,7 @@ internal static void ValidateFormatArgs(string name, CheckTypesContext checkType
                         HasNumericFmt = false
                     };
 
-                    if (!TextFormatUtils.IsValidFormatArg(formatArg, null, out textFormatArgs))
+                    if (!TextFormatUtils.IsValidFormatArg(formatArg, null, null, out textFormatArgs))
```

Comment:
>null, null [](http://example.com/codeflow?start=69&length=10)

Can you use named arguments to make it more readable?

Reply:
updated